language/German/strings.po

```diff
@@ -261,7 +261,7 @@ msgstr "Schieberegler"
 
 #: src/emu/ui/custui.cpp:541
 msgid "Gfx viewer background"
-msgstr "Hintergrund Gfx Anzeige"
+msgstr "Hintergrund Gfx-Anzeige"
```

Comment:
Maybe we can translate "Gfx" with "Grafik" or something else? For me, it's pretty unclear what this option does...